docs/reference/dropbox.md

```diff
@@ -0,0 +1,118 @@
+
+2. Once the app is created, please store **app key** and **app secret** for later configuration use.
+
+### Generate a Refresh Token
+
+To generate a refresh token, follow the next steps:
+1. Hit the following URL with the generated APP_KEY on your browser:
+    ```shell
+    https://www.dropbox.com/oauth2/authorize?client_id=<APP_KEY>&response_type=code&token_access_type=offline
+    ```
+    
+    In the HTTP response, you will get an **authorization code** that will be used to generate a refresh token.
+
+    **Note:** Authorization code can only be used once to create a refresh token.
+
+2. Perform the following POST API call in the terminal:
+    ```shell
+    curl -X POST "https://api.dropboxapi.com/oauth2/token?code=<AUTHORIZATION_CODE>&grant_type=authorization_code" -u "<APP_KEY>:<APP_SECRET>"
+    ```
+    Store the refresh token from the response and use it in the connector configuration.
+    **Note:** Make sure the response has a similar list of following scopes:
+    - account_info.read
+    - files.content.read
+    - files.metadata.read
+    - sharing.read
+
+## Configuration
+
+### Configure Dropbox connector
+
+The following configuration fields need to be provided for setting up the connector:
+
+#### `path`
+
+The folder path to fetch files/folders for Dropbox connector. Default value is `/`.
+
+#### `app_key`
+
+The App Key to authenticate the Dropbox application.
+
+#### `app_secret`
+
+The App Secret to authenticate the Dropbox application.
+
+#### `refresh_token`
+
+The Refresh Token to authenticate the Dropbox application.
+
+#### `retry_count`
+
+The number of retry attempts after failed request to Dropbox. Default value is `3`.
+
+#### `concurrent_downloads`
+
+The number of concurrent downloads for fetching the attachment content. This might speed up the content extraction of attachments. Defaults to `100`.
+
+### Content Extraction
+
+The connector uses the Elastic ingest attachment processor plugin for extracting file contents. The ingest attachment processor extracts files by using the Apache text extraction library Tika. Supported file types eligible for extraction can be found as `TIKA_SUPPORTED_FILETYPES` in [utils.py](../connectors/utils.py) file.
+
+## Documents and Sync
+
+The connector syncs the following objects and entities:
+- **Files**
+    - Includes metadata such as file name, path, size, content, etc.
+- **Folders**
+
+## Sync rules
+
+- Files bigger than 10 MB won't be extracted
+- Permissions are not synced. **All documents** indexed to an Elastic deployment will be visible to **all users with access** to that Elastic Deployment.
+
+## Connector Client operations
+
+### End-to-end Testing
+
+End-to-end is not a part of the current version of the connector.
+
+## Known issues
+
+- There is a delay in the updated metadata of paper files to be reflected in the UI hence response does not have updated results. As soon as the change is reflected in the UI, the same is available in response.
```

Comment:
This isn't super clear to me!

Is this a correct rephrasing?

_There is a known issue where metadata updates to Paper files are not immediately reflected in the Dropbox UI, which delays the availability of updated results for the connector. Once the metadata changes are visible in the Dropbox UI, the updates are available._

IIUC I guess this is a problem on the Dropbox side?

Reply:
Yes, this is a problem on the Dropbox side. And the suggested rephrasing seems good to me 👍